crates/top/re_sdk/src/recording_stream.rs

```diff
@@ -695,6 +644,10 @@ impl RecordingStreamBuilder {
     /// ```
     //
     // # TODO(#5531): keep static data around.
+    #[deprecated(
+        since = "0.24.0",
+        note = "Use `rec.serve_grpc()` + `rerun::serve_web_viewer()` instead"
```

Comment:
This could link to `docs/snippets/all/howto/serve_web_viewer.rs` or docs/content/reference/migration/migration-0-24.md